### PR TITLE
Improvement/updated policies sudden change

### PR DESCRIPTION
**Pull request by jemassey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ For detailed information on this package, please refer to the [online documentat
 
 ### Version next
 
+### Version 1.3.1
```
Comment (TheConnMan) [inline README.md]: The version is only added when dev is cut over to master.

File: package.json [hunk 2]
```diff
@@ -1,39 +1,63 @@
...
-  "id" : "netuitive.packages.aws.alb",
-  "version" : "1.3.0",
-  "manifest" : {
-    "name" : "Netuitive Packages AWS ALB",
-    "description" : "A set of Netuitive analytics configurations, polices, dashboards, and reports that are used to monitor performance of AWS Application Load Balancer (ALB) resources.",
-    "author" : {
-      "name" : "Netuitive Research",
-      "email" : "research@netuitive.com"
+  "id": "netuitive.packages.aws.alb",
+  "version": "1.3.1",
```
Comment (TheConnMan) [inline package.json]: Same as in the readme.

File: policies/decreased_load_balancer_error_rate.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/decreased_load_balancer_error_rate.json]: These are internal IDs. Please remove them.

File: policies/decreased_load_balancer_error_rate.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      "level": 10.0,
+      "metric": "netuitive.aws.alb.httpcodeelberrorpercent",
+      "operator": "<"
+    }],
+    "creatorEmail": "research@netuitive.com",
+    "deleted": false,
+    "description": "This is an error rate policy looking at errors from the Load Balancer itself.  Looking for a sudden change in error rates that exceed 10%. In those cases, a Critical event will be generated. You may wish to tune percentage increase threshold to better suit your environment.",
+    "duration": 300,
+    "enabled": true,
+    "id": "4ccd5f3f-f42e-3f8b-a1ef-7921639cd97d",
```
Comment (TheConnMan) [inline policies/decreased_load_balancer_error_rate.json]: IDs and last updated should be removed.

File: policies/decreased_request_count.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/decreased_request_count.json]: See above.

File: policies/decreased_target_response_time.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/decreased_target_response_time.json]: See above.

File: policies/increased_load_balancer_error_rate.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/increased_load_balancer_error_rate.json]: See above.

File: policies/increased_request_count.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/increased_request_count.json]: See above.

File: policies/increased_target_response_time.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
+{
+  "policy": {
+    "actions": [{
+        "category": 3,
+        "type": "event"
+      },
+      {
+        "enabled": true,
+        "id": 54829,
```
Comment (TheConnMan) [inline policies/increased_target_response_time.json]: See above.

**Review by TheConnMan — CHANGES_REQUESTED** [review]

**Review by TheConnMan — APPROVED** [review]

**Review by Grumblesmith — APPROVED** [review]

File: package.json [hunk 10]
```diff
@@ -1,39 +1,57 @@
...
-  "id" : "netuitive.packages.aws.alb",
-  "version" : "1.3.0",
-  "manifest" : {
-    "name" : "Netuitive Packages AWS ALB",
-    "description" : "A set of Netuitive analytics configurations, polices, dashboards, and reports that are used to monitor performance of AWS Application Load Balancer (ALB) resources.",
-    "author" : {
-      "name" : "Netuitive Research",
-      "email" : "research@netuitive.com"
+  "id": "netuitive.packages.aws.alb",
+  "version": "1.4.0",
```
Comment (TheConnMan) [inline package.json]: The version should not be updated in PRs.

**Review by TheConnMan — CHANGES_REQUESTED** [review]

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**